po/de.po

```diff
@@ -128,7 +164,23 @@ msgstr ""
 "%s ist ein Programm zum Umsortieren und Modifizieren von PDF-Dokumenten. "
 "Geschrieben in GTK+ und Python."
 
-#: pdfarranger/pdfarranger.py:1449
+#: pdfarranger/pdfarranger.py:1631
+msgid "Maintainers and contributors"
+msgstr "Verwalter und Mitwirkende"
```

Comment:
I would prefer Maintainer here. 
https://de.m.wikipedia.org/wiki/Maintainer

Reply:
I don't see a reason to use English here. But "Verwalter" isn't nice, I agree.
Why not Entwickler or Hauptentwickler?

Reply:
Hauptentwickler is a good idea! In context, I'd say going with "Entwickler" (plural) only would actually be the nicest.

Reply:
OK, I'll change it to Entwickler.